.circleci/config.yml

```diff
@@ -92,6 +92,43 @@ jobs:
       - continuation/continue:
           configuration_path: .circleci/generated_config.yml
 
+  # Barretenberg
+  barretenberg-wasm-linux-clang:
```

Comment:
This order was changed as we now portal `bb.js` in `noir_js_backend_barretenberg` in order to get CI fully passing on `aztec-packages`. We just will not be able to sync to Noir until we do a bb.js release